.github/ISSUE_TEMPLATE.md

```diff
@@ -17,8 +17,20 @@ New Feature Request / Improvement:
 1. Should let user see such and such
 
 Bug:
-- Please provide steps to reproduce including browser version
+- Use 'Bug' label
```

Comment:
unless they've been invited to the organization, outside contributors will not be able to change any issues fields, fyi

Reply:
@saimatsumoto 
Did you plan on re-wording or removing?
>  +- Use 'Bug' label

since
> outside contributors will not be able to change any issues fields

Reply:
I can remove it. @thescientist13